thorlcr/mfilemanager/thmfilemanager.cpp

```diff
@@ -476,7 +476,9 @@ class CFileManager : public CSimpleInterface, implements IThorFileManager
             desc->queryProperties().setProp("@workunit", wuidStr.str());
             desc->queryProperties().setProp("@job", jobStr.str());
             desc->queryProperties().setProp("@owner", userStr.str());
-            if (job.getOptBool("subDirPerFilePart"))
+
+            // default to directory per part in containerized mode only for now
+            if (job.getOptBool("subDirPerFilePart", isContainerized()))
```

Comment:
I wonder if this should also take into account on optional flag on a storage plane. (Defaulting as you have it here.)

Reply:
@ghalliday - please see new commit, have changed so checking plane property and defaulting there to on.